crates/miden-lib/src/transaction/mod.rs

```diff
@@ -197,23 +198,27 @@ impl TransactionKernel {
     /// ```text
     /// [
     ///     expiration_block_num,
+    ///     ACCOUNT_UPDATE_COMMITMENT,
     ///     OUTPUT_NOTES_COMMITMENT,
```

Comment:
This was incorrect before, right? In any case, if this illustrates the stack should it be the other way around?